composer/utils/eval_client/eval_client.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Should this be in utils...? Thoughts @dakinggg @eracah. I guess consistent w objectstore, but imo that should be at top level and not utils